### PR TITLE
Swope errata6 (in review)

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by anthonyfister — COMMENTED** [review]
Note that we are supporting OCP 4.8 with 2.2.6 and later... **BUT** we have a **known issue** that  **support of OCP 4.8 on Bare Metal is not working** (issue #13183).

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -9,6 +9,32 @@ By default, errata updates are automatically applied when released. See link:../
...
 
+== Errata 2.2.6
+
+*Important:* See information in the previous notes.
+
+{ocp} 4.8 is supported with 2.2.6 and later. 
+
+View a summarized list of fixes that the {product-title} Errata provides:
+
+* Fixes issues with imported clusters. Imported clusters from {ocp-short} 3.11 in metrics visibility appear as other {ocp-short} appear. (GitHub 13162)
```
Comment (cadawson) [inline release_notes/errata.adoc]: Fixes issues with imported clusters. Imported clusters from {ocp-short} 3.11 in metrics visibility appear as other {ocp-short} ?clusters? appear.

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -9,6 +9,32 @@ By default, errata updates are automatically applied when released. See link:../
...
+
+* Fixes deployment failure with an application that subscribes to a Helm chart from a Git repository server with a custom CA certificate. Configure the application channel with either `insecureSkipVerify: true` to avoid custom CA verification, or add the custom CA certificates to the channel configuration. (GitHub 14467)
+
+* Updates redisgraph StatefulSet for metadata and specification updates to fix upgrade issues. (GitHub 13661)
```
Comment (cadawson) [inline release_notes/errata.adoc]: RedisGraph `StatefulSet`

**Review by cadawson — COMMENTED** [review]
I know I am early, but I hope it helps to get a review in.

File: release_notes/errata.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/errata.adoc]: I had not put this in review bc I was literally still working this all. WIP in the PR.

File: release_notes/errata.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/errata.adoc]: I had not put this in review bc I was literally still working this all. WIP in the PR.

Comment (swopebe) [reply release_notes/errata.adoc]: I will bring you in when I am done.

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -9,6 +9,32 @@ By default, errata updates are automatically applied when released. See link:../
 
 - For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user.
 
+== Errata 2.2.6
+
+*Important:* See information in the previous notes.
+
+{ocp} 4.8 is supported with 2.2.6 and later. *Important: {ocp} 4.8 does not work for this release.
```
Comment (swopebe) [inline release_notes/errata.adoc]: @anthonyfister that is documented here ^^^^ can you take a look?

File: release_notes/errata.adoc [hunk 4]
```diff
@@ -9,6 +9,32 @@ By default, errata updates are automatically applied when released. See link:../
...
+* Fixes the compare and sort functionality of the Configuration Policy Controller to stop a continual increase of the Kubernetes resource on a managed cluster that is configured by the policy controller. (https://bugzilla.redhat.com/show_bug.cgi?id=1973772[Bugzilla 1973772])
+
+* Updates the configuration policy namespace selection to identify the policy as `NonCompliant` when any selected namespace is `NonCompliant`. (https://bugzilla.redhat.com/show_bug.cgi?id=1969845[Bugzilla 1969845])
+
+* Removes the use of the `SelfLink` attribute in `cert-manager` for compatibility with {ocp-short} 4.8. (GitHub 13121)
+
+* Fixes an issue with `SecurityConstraintContext` policy, which prevented the search operator pod from starting. Updates the Docker image to run with a non-root user by default and updates the security context in the search operator deployment with `runAsNonRoot`, which starts the container with a non-root user. (https://bugzilla.redhat.com/show_bug.cgi?id=1967953[Bugzilla 1967953])
+
+* Adds various container updates.
+
```
Comment (swopebe) [inline release_notes/errata.adoc:35]: @anthonyfister this is the placeholder for the one I am missing.

Comment (swopebe) [reply release_notes/errata.adoc:35]: came up with my own general statement that should work

**Review by cadawson — APPROVED** [review]
Looks good! Thank you!

File: release_notes/errata.adoc (hunk 4 above)
Comment (anthonyfister) [reply release_notes/errata.adoc:35]: Looks good

**Review by anthonyfister — APPROVED** [review]
This looks good to me.

**Pull request merged**